codelabs/solutions/kayenta/README.md

```diff
@@ -0,0 +1,9 @@
+# Automated canary analysis on Kubernetes Engine with Spinnaker
```

Comment:
Any reason these are in the solutions/ subdirectory?

Reply:
It was suggested by @duftler. I can move it if needed.
It's not a codelab though.

Reply:
After talking about it with @lwander a bit, I think best approach is to have `solutions` as a sibling to `codelabs`.

Reply:
Moved everything to the root of the repo.